src/styles/card.scss

```diff
@@ -0,0 +1,130 @@
+* {
```

Comment:
why does card file contains `*` font-face connection and .container?
create separate file for them